src/wp-includes/http.php

```diff
@@ -581,7 +581,21 @@ function wp_http_validate_url( $url ) {
 	}
 
 	$port = $parsed_url['port'];
-	if ( 80 === $port || 443 === $port || 8080 === $port ) {
+
+
+	/**
+	 * Controls list of ports considered safe in HTTP Api
+	 *
+	 * Allows to change and allow external requests for the HTTP request.
+	 *
+	 * @since 5.9.0
+	 *
+	 * @param array  $allowed_porst Array of integers for valid ports
```

Comment:
Corrected typo and added period to the end of the comment for consistency:
```suggestion
	 * @param array  $allowed_ports Array of integers for valid ports.
```